modules/sdk-core/src/bitgo/tss/eddsa/index.ts

```diff
@@ -9,3 +9,5 @@ export {
   sendSignatureShare,
   encryptYShare,
 } from './eddsa';
+
+export { EncryptedYShare, DecryptableYShare, CombinedKey, SigningMaterial } from './types';
```

Comment:
We can do something like so that we can properly differentiate between EDDSA and ECDSA things
```suggestion
export * as EDDSAMethodTypes from './types';
```